.flake8

```diff
@@ -9,6 +9,7 @@ exclude=
     .setup/bin/reupload_generate_csv.py,
     .setup/bin/reupload_old_assignments.py,
     .setup/bin/setup_sample_courses.py,
+    .setup/bin/sample_courses/models,
```

Comment:
Let's ignore all of the individual files rather than an entire directory.  That way, if someone adds a new file, it will get linted.